set.mm

```diff
@@ -423711,7 +423723,7 @@ version of it and the OLD version is obsolete (and will be removed
        variable replaced by a disjoint variable condition "v" with a
        conjunction as antecedent "a").  As a general rule, the suffixes for
        the theorem forms ("i", "d" or "g") should be the first of multiple
-       suffixes, as for example in ~ vtocldf or ~ rabeqif .
+       suffixes, as for example in ~ vtocldf or ~ rabeqi .
```

Comment:
This is not an example for this case anymore!

Reply:
I added the commit suggestions in a bit of a hurry, I'll take some time to double check again.